apps/avifenc.c

```diff
@@ -1898,6 +1906,15 @@ int main(int argc, char * argv[])
         ++argIndex;
     }
 
+    // Alpha quality defaults to the same value as (color) quality until the first time it's explicitly set.
+    for (int i = 0; i < input.filesCount; ++i) {
+        avifInputFileSettings * fileSettings = &input.files[i].settings;
+        if (fileSettings->qualityAlpha.set) {
+            break;
+        }
+        fileSettings->qualityAlpha = fileSettings->quality;
+    }
+
```

Comment:
Maryla: Thank you for making this change! I tried to review it but found that the code in avifenc that handles the command-line options has become very complicated.

I am wondering if the deprecated --minalpha and --maxalpha options should also default to the deprecated --min and --max options. But I can't figure out the default values for these four options.

Reply:
I agree the code is quite complex!
I double checked and this change does work for --min --max too. If --minalpha and --maxalpha are not specified, then the alpha quality will be the same as the color quality. This can be seen in `test_cmd.sh`:

```shell
  "${AVIFENC}" -s 10 --min 15 --max 25 "${INPUT_Y4M}" "${ENCODED_FILE}" > "${OUT_MSG}"
  grep " color quality \[68 " "${OUT_MSG}"
  grep " alpha quality \[68 " "${OUT_MSG}"
```

Reply:
Thanks for looking into it. With `--min 15 --max 25`, we will use their midpoint 20 as the quantizer, and convert the quantizer to quality with the formula `quality = ((63 - quantizer) * 100 + 31) / 63`. So we will get quality 68, which matches your test.